.travis.yml

```diff
@@ -41,7 +41,7 @@ env:
     - COOKIECUTTER_DATABASE=mysql COOKIECUTTER_ELASTICSEARCH=elasticsearch6 ES_VERSION=6.2.2
 
 python:
-  - "3.6"
+  - "3.7"
```

Comment:
@ChiaraBi Looks like we won't be able to ship 3.7, see https://github.com/celery/celery/issues/4500.